extensions/mssql/package.json

```diff
@@ -557,7 +557,13 @@
           "description": "%mssql.connectionOptions.authType.description%",
           "groupName": "Security",
           "valueType": "category",
-          "defaultValue": null,
+          "defaultValue": "SqlLogin",
+          "defaultValueOsOverrides": [
+            {
+              "osEnumValue": 1,
```

Comment:
I would use actual strings here ('darwin', 'win32', 'linux')

Reply:
Let me check the OS enum. If there is an easy way to compare the value as a string, I will do that.

Reply:
Done. String values are Windows, Macintosh, and Linux, which matches the OperatingSystem enum.